components/Navbar.tsx

```diff
@@ -1,8 +1,14 @@
+'use client';
```

Comment:
Why this?

Reply:
This is a new upcoming Next.js feature called server actions.

"use client" is likely a directive or annotation used to explicitly indicate that a function is intended for use on the client side of your application. By marking the function with "use client", you're informing the environment or framework that the function is safe to be passed directly to client components

Reply:
Yes, but why are we adding it here? Does it do anything, can we achieve the same result without it?